### PR TITLE
Better readme in stock_mts_mto_rule

### PR DESCRIPTION
**Pull request by gabrielo77**
Fix issue #83

### CODE REVIEW

File: stock_mts_mto_rule/README.rst [hunk 1]
```diff
@@ -37,6 +37,7 @@ Usage
 
 You have to choose the mts+mto route on the product form.
 You should not choose both mts+mto route and mto route.
+You have to choose 'Use MTO+MTS rules' on the company's warehouse form.
```
Comment (pedrobaeza) [inline stock_mts_mto_rule/README.rst:44]: Please put it on a "Configuration" section below Usage section

Comment (gabrielo77) [inline stock_mts_mto_rule/README.rst:44]: > Please put it on a "Configuration" section below Usage section

## Done!

Gabriel.

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by tafaRU** [comment]
@gabrielo77,  the git author of your commit seems not properly recognized, could you please check your configuration?
The rest LGTM :+1:

**Comment by gabrielo77** [comment]
On 23 September 2015 at 13:19, Alex Comba notifications@github.com wrote:

> @gabrielo77 https://github.com/gabrielo77, the git author of you commit
> seems not properly recognized, could you please check your configuration?
> The rest LGTM [image: :+1:]

Yeah, my email was misconfigured, now it is ok.

## 

Gabriel.

**Comment by tafaRU** [comment]
@gabrielo77, please also consider to squash all commits in one.
Thank you in advance!

**Comment by florian-dacosta** [comment]
:+1:

**Comment by gabrielo77** [comment]
> @gabrielo77 https://github.com/gabrielo77, please also consider to
> squash all commits in one.
> Thank you in advance!
> 
> Ok, I'll create a new PR and close this one.

Thank you too!

## 

Gabriel.

**Pull request closed without merging**